collections/_article/progress-report-web-export-in-4-3.md

```diff
@@ -0,0 +1,311 @@
+solved as many bugs as I did. Please try it out! The audio work, in particular, sounds incredible.
+
+### Here be dragons
+Even if the odds are that sample playback will be a part of Godot 4.3 release, it was developed very quickly,
+and even if it works pretty well, if you ask me, it still is pretty much a prototype. But we can't just release Godot 4.3
+without proper audio support for single-threaded releases.
+
+So it has been decided that it should be treated as _experimental_. More work will be done before the release of Godot 4.4
+in order to stabilize the API and how it works under the hood.
+
+### Call to action
+But even before thinking about Godot 4.4, I must call for your help! We need people to test out [the sample playback <abbr title="Pull Request">PR</abbr>](https://github.com/godotengine/godot/pull/85939)
+until the release.
+
+If you encounter any issue while testing my PR or while testing out the single-threaded samples builds, please report
+them [here](https://github.com/godotengine/godot/pull/85939).
+
+Finally, if you still intend to continue using threads on the Web platform, you can! And the next feature should aid you
+setup your game online.
+
+## Easy <abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> <abbr title="Progressive Web App">PWA</abbr> for threaded builds
+With this new feature, people are able to activate the necessary <abbr title="Cross-Origin-Opener-Policy">COOP</abbr>/
+<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers without having access to the server. You'll need to
+export your game as a Progressive Web App though.
+
+![Ensure Cross-Origin Isolation Headers export option](/storage/blog/progress-report-web-export-in-4-3/ensure-cross-origin-isolation-headers.webp)
+
+This is because this export option setup the Service Worker, included with the <abbr title="Progressive Web App">PWA</abbr>
+build, to intercept the requests from the server and inject those missing
+<abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers.
+
+## Boot splash screens
+Just before the beta feature freeze deadline, [@patwork](https://github.com/patwork), a new
+[#web](https://chat.godotengine.org/channel/web) contributor,
+[managed to sneak up something really nice](https://github.com/godotengine/godot/pull/91128):
+
+He cleaned up the game page code and he updated the progress bar to use the semantic (and very useful) HTML `<progress>`
+tag.
+
+But, mainly and from now on, he made boot splash screens set in the project settings to be displayed on the screen
+while the engine and the assets are getting loaded by the browser.
+
+![New Godot Web loading screen](/storage/blog/progress-report-web-export-in-4-3/loading-screen.gif)
+
+If no boot splash screen has been set, the classic Godot boot splash will be seen, as it is currently displayed on
+other platforms.
```

Comment:
This is the only change that does not pertain to audio - in case we want to shorten the post further, I would remove and change the title to specifically state Web Audio as the topic

Reply:
No, the single-threaded web export for Godot is not audio-related per se.

Reply:
Fair, then language review:

```suggestion
## Boot splash screens
Just before the beta feature freeze deadline, [@patwork](https://github.com/patwork), a new
[#web](https://chat.godotengine.org/channel/web) contributor,
managed to include [some neat features](https://github.com/godotengine/godot/pull/91128):

He cleaned up the game page code and he updated the progress bar to use the semantic (and very useful) HTML `<progress>`
tag.

But, mainly and from now on, he made boot splash screens set in the project settings to be displayed on the screen
while the engine and the assets are getting loaded by the browser.

![New Godot Web loading screen](/storage/blog/progress-report-web-export-in-4-3/loading-screen.gif)

If no boot splash screen has been set, the classic Godot boot splash will be seen, as it is currently displayed on
other platforms.
```